subworkflows/local/bgc.nf

```diff
@@ -0,0 +1,100 @@
+/*
+    Run AMP screening tools
+*/
+include { UNTAR as UNTAR_CSS                       } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_DETECTION                 } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_MODULES                   } from '../../modules/nf-core/modules/untar/main'
+include { ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES } from '../../modules/nf-core/modules/antismash/antismashlitedownloaddatabases/main'
+include { ANTISMASH_ANTISMASHLITE                  } from '../../modules/nf-core/modules/antismash/antismashlite/main'
+
+workflow BGC {
+
+    take:
+    fna     // tuple val(meta), path(PROKKA.out.fna)
+    gff     // tuple val(meta), path(PROKKA.out.gff)
+
+    main:
+    ch_versions = Channel.empty()
+    ch_mqc      = Channel.empty()
+
```

Comment:
Add comment for future devs about lkocally testing all combinations if anything to do with antismash changes

Reply:
See line [21](https://github.com/nf-core/funcscan/pull/61/files#diff-3433c51d840aa1e4c103994e80b1ece2f236bac58d54cd8d8589f867162a0f19R21)